conda_build/scripts.py

```diff
@@ -37,7 +37,7 @@ def iter_entry_points(items):
 
 
 def create_entry_point(path, module, func):
-    pyscript = PY_TMPL % (module, func, func)
+    pyscript = PY_TMPL % {'module': module, 'func': func}
```

Comment:
:smile: - I don't disagree, but am not convinced it was necessary to support entry points. Just describing the rationale in the description would be good enough for me :+1: